apps/docs/content/guides/platform/compute-add-ons.mdx

```diff
@@ -4,7 +4,7 @@ title: 'Compute Add-ons'
 description: "Learn about your project's instance and additional add-ons."
 ---
 
-Every project on the Supabase Platform comes with its own dedicated Postgres instance running inside a virtual machine (VM). The following table describes the base instance with additional compute add-ons available if you need extra performance when scaling up Supabase.
+Every project on the Supabase Platform comes with its own dedicated Postgres instance running inside a virtual machine (VM). The following table describes the base add-ons with additional compute add-ons available if you need extra performance when scaling up Supabase.
```

Comment:
this one's OK to retain as 'instance', as it's referring to the base level included with the plan vs the add-ons (which are mentioned as `additional compute add-ons`